Apps/AdminWebClient/src/ClientApp/src/services/restCommunicationService.ts

```diff
@@ -7,7 +7,7 @@ import { ResultType } from "@/constants/resulttype";
 
 @injectable()
 export class RestCommunicationService implements ICommunicationService {
-  private readonly BASE_URI: string = "v1/api/Communication";
+    private readonly BASE_URI: string = "v1/api/Communication";
```

Comment:
see spacing issue mentioned above

Reply:
This should have been fixed with my most recent commit today.